README.md

```diff
@@ -7,52 +8,48 @@ A dockless mobility data API built with Python/Sanic
 
 The source database for the API is a modified geojson file. TODO: add sample data and link to processing tools to generate source data.
 
-### Option 1: Run w/ Python 3
+### Option 1: Run w/ Docker (Suggested)
 
-1. Clone repo and `cd` into it.
+1.  Install [docker](https://www.docker.com/) and start the engine:
+    `systemctl start docker`
 
-2. Install python requirements:
-```python
-pip install -r requirements.txt
-```
-3. Install [libspatialindex](http://libspatialindex.github.io/)
+2.  Clone repo and `cd` into it.
+    `git clone https://github.com/cityofaustin/dockless-api.git`
 
-4. Copy `grid.json` source data to `../dockless-api/app/data`
+3.  Copy `grid.json` source data to `../dockless-api/app/data`
 
-5. Start the server:
-```python
-python app/app.py
-```
+4.  Start the docker server (in the background on port 80)
+
+`./scripts/serve-local.sh`
```

Comment:
Create a docker a folder with bash script files for different environments. I think this will be easier than documenting the script in the readme and this follows the pattern that @benweatherman setup in the Janis and Joplin repos.

The deploy file is current just a blank placeholder.

💕 missu Ben 💕

Reply:
Glad to see my legacy lives on

Reply:
yes, you're still *serving locals*